pkg/manager/member/tiflash_member_manager.go

```diff
@@ -686,6 +686,9 @@ func (tfmm *tiflashMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster
 		return err
 	}
 	for _, store := range tombstoneStoresInfo.Stores {
+		if store.Store != nil && !pattern.Match([]byte(store.Store.Address)) {
+			continue
+		}
```

Comment:
I think we should add an unit test here to ensure the syncing status would only add the target store.

Reply:
updated